.github/workflows/test-lang-py-ARM.yml

```diff
@@ -21,7 +21,7 @@ on:
   pull_request:
     branches: [ main ]
     paths:
-    - .github/workflows/test-lang-py.yml
+    - .github/workflows/test-lang-py-ARM.yml
```

Comment:
The ARM64 VMs are self-hosted by ASF Infra and they use Ubuntu 20.04 with an old Python version in the OS repos ...
This should be re-worked to use Conda or something similar to install a newer version of Python.